### PR TITLE
Add population and popularity

### PR DESCRIPTION
**Pull request by trescube**
pull `gn:population` and `misc:photo_sum` from source WOF record for popluation and popularity, respectively

Fixes #29

### CODE REVIEW

**Comment by trescube** [comment]
Fixed #29

File: package.json [hunk 1]
```diff
@@ -31,6 +31,7 @@
     "pelias-dbclient": "0.0.9",
     "pelias-logger": "0.0.8",
     "pelias-model": "^1.0.0",
+    "temp": "^0.8.3",
```
Comment (dianashk) [inline package.json]: I think this should go into `devDependencies`

**Comment by dianashk** [comment]
:+1:

File: package.json (hunk 1 above)
Comment (trescube) [inline package.json]: D'oh!  Good call.

**Comment by trescube** [comment]
Merged

**Pull request merged**